Reference/Cache/updating-cache-v9.md

```diff
@@ -0,0 +1,46 @@
+---
+versionFrom: 9.0.0
+
+---
+
+# Getting/Adding/Updating/Inserting Into Cache
+
+_This section describes how you should be getting/adding/updating/inserting items in the cache._
+
+## Adding and retrieving items in the cache
+
+The recommended way to put data in and get data out is to use one of the many overloaded methods of: `GetCacheItem`. The `GetCacheItem` methods (all except one) are designed to "Get or Add" to the cache. For example, the following will retrieve an item from the cache and if it doesn't exist will ensure that the item is added to it:
+
+```csharp
+MyObject cachedItem = _appCaches.RuntimeCache.GetCacheItem<MyObject>("MyCacheKey", () => new MyObject());
+```
+
+where `_appCaches` is injected as type `AppCaches`.
+
+Notice 2 things:
+
+* The `GetCacheItem` method is strongly typed and
+* We are supplying a callback method which is used to populate the cache if it doesn't exist.
+
+The example above will retrieve a strongly typed object of `MyObject` from the cache with the key of "MyCacheKey", if the object doesn't exist in the cache a new instance of `MyObject` will be added to it with the same key.
+
+There are a couple of overloads of `GetCacheItem` allowing you to customize how your object is cached from cache dependencies to expiration times.
+
+To use this generic implementation, add the `Umbraco.Extensions` namespace to your code.
+
+### Retrieving an item from the cache without a callback
+
+One of the overloads of `GetCacheItem` doesn't specify a callback, this will allow  you to retrieve an item from the cache without populating it if it doesn't exist.
```

Comment:
```suggestion
One of the overloads of `GetCacheItem` doesn't specify a callback, this will allow you to retrieve an item from the cache without populating it if it doesn't exist.
```